packages/react/src/utils/index.ts

```diff
@@ -1,4 +1,7 @@
-export { unknownString } from './const';
+/**
+ * @deprecated Will be removed in the future. Use `unknownString` from `@grafana/faro-core` instead.
```

Comment:
I still keep the global export just in case users do use it for some reasons 👍 
Maybe we are also good with removing the export from the faro-react package 🤔